unlock-app/src/utils/priceFormatter.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
```

Comment:
Please move to the `ui` package and add storybook stories. https://storybook.js.org/